CI/src/main/java/group1/email.java

```diff
@@ -0,0 +1,83 @@
+package group1;
+
+/*
+https://netcorecloud.com/tutorials/send-email-in-java-using-gmail-smtp/?hss_channel=tw-3300992070
+
+This code is not writen by group 1 it is copied line by line from the link abouve.
+Group 1 does not clame to have writen this code.
+*/
+
+
+import java.util.Properties;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.PasswordAuthentication;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+public class email {
+
+    public static void main(String[] args) {
```

Comment:
Please make this into a static function that takes a message, subject line and recipient as arguments.

Reply:
And also please add tests to test the email functionality if possible.